### ISSUE TITLE
Write up "Begin Your Adventure" guide

### ISSUE THREAD

**Issue by zkat** [iron-vault-plugin/iron-vault#228]
including an initial scene to show off how to play with mechanics blocks.

**Comment by RaulMarq** [comment]
Looking forward to when this is completed, been looking foward to trying Iron Vault at some point and it'd be great to have what I assume is the final part of the guide!

**Comment by Newman5** [comment]
Yep!  I got to the end of the Player's guide and ... was a bit disappointed.  No worries because I feel like I can take it from here. My next steps are to watch some playthru videos on youtube.  Got any recommendations?  I'm looking for something that really focuses on the mechanics of Ironsworn (playing with Iron Vault would be great, but perhaps too much to ask)?

With deep gratitude for this wonderful resource, Newman
